huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -522,6 +527,10 @@ func resourceRdsInstanceCreate(ctx context.Context, d *schema.ResourceData, meta
 		}
 	}
 
+	if err := updateRdsInstanceBackupStrategy(d, client, instanceID); err != nil {
+		return diag.FromErr(err)
```

Comment:
it's better to throw a warning rather than return an error.

Reply:
This function is like this when UpdateInstance,Is it ok to just warn when CreateInstance?